Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [DNSNet/App](https://hosted.weblate.org/projects/dnsnet/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/dnsnet/app/horizontal-auto.svg)

lucasmz-dev commented:
@t895 hey can you commit the weblate changes that weren't put into a commit yet, wanna do a last review while I have energy, basically

edit: thanks :heart:

t895 commented:
@lucasmz-dev Just committed
Thank you for your contributions!

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (234 of 234 strings)

Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/dnsnet/app/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/dnsnet/metadata/pt_BR/
Translation: DNSNet/App
Translation: DNSNet/Metadata

weblate committed:
Update translation files

Updated by "Cleanup translation files" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/dnsnet/app/
Translation: DNSNet/App

t895 committed:
Translated using Weblate (Russian)

Currently translated at 89.7% (210 of 234 strings)

Translated using Weblate (Russian)

Currently translated at 89.7% (210 of 234 strings)

Co-authored-by: Charles Lombardo <clombardo169@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/dnsnet/app/ru/
Translation: DNSNet/App

Commit:
Added translation using Weblate (Chinese (Traditional Han script))

Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>

Commit:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 76.0% (178 of 234 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (2 of 2 strings)

Co-authored-by: hugoalh <hugoalh@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/dnsnet/app/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/dnsnet/metadata/zh_Hant/
Translation: DNSNet/App
Translation: DNSNet/Metadata

weblate committed:
Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/dnsnet/app/
Translation: DNSNet/App
